wix/main.wxs

```diff
@@ -21,11 +21,9 @@
   destination for 32-bit versus 64-bit installers. Removal of these lines will
   cause installation errors.
 -->
-<?if $(var.Platform) = x64 ?>
-    <?define Win64 = "yes" ?>
+<?if $(sys.BUILDARCH) = x64 or $(sys.BUILDARCH) = arm64 ?>
     <?define PlatformProgramFilesFolder = "ProgramFiles64Folder" ?>
 <?else ?>
-  <?define Win64 = "no" ?>
```

Comment:
Why the `Win64` changes here?

Reply:
The `Win64` and `Platform` settings have been removed as they are now deprecated in wix. The details are described in the following pull request from cargo-wix, which is shared here.

https://github.com/volks73/cargo-wix/pull/113#issue-723263542

>I took the opportunity to remove use of Win64 and Platform in the wxs: those attributes are deprecated according to the wix documentation, passing -arch on the command line should be prefered (which this PR now does). var.Platform is replaced by sys.BUILDARCH.